radsat-sk/framework/serial/RUart.c

```diff
@@ -48,15 +48,15 @@ static int initialized[UART_BUS_COUNT] = {0};
  */
 int uartInit(UARTbus bus) {
 
-	// only allow initialization once
-	if (initialized[bus] != 0)
-		return E_IS_INITIALIZED;
+	// only allow initialization once (exit gracefully)
+	if (initialized[bus])
+		return 0;
```

Comment:
Curious about your logic for removing the E_IS_INITALIZED

Reply:
Wanted graceful returns since there's no actual reason to throw an error; the caller wants it initialized, and it already was, so just return with a smile :)